helm/robusta/values.yaml

```diff
@@ -511,7 +511,7 @@ kube-prometheus-stack:
         - name: 'null'
         - name:  'robusta'
           webhook_configs:
-            - url: 'http://robusta-runner.{{ .Release.Namespace }}.svc.cluster.local/api/alerts'
+            - url: 'http://robusta-runner.{{ .Release.Namespace }}/api/alerts'
```

Comment:
@yahya-hrln Can you please make it configurable, so it won't change the default behavior for all users?
i.e., put the DNS suffix in a helm value, defaulted to `.svc.cluster.local`

My concern is, that some users have defined network policies for the existing url, and this change might break it for them